app/src/main/java/com/jasonette/seed/Action/JasonUtilAction.java

```diff
@@ -26,18 +22,26 @@
 import android.widget.TimePicker;
 import android.widget.Toast;
 
-import com.github.florent37.singledateandtimepicker.dialog.SingleDateAndTimePickerDialog;
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+import androidx.localbroadcastmanager.content.LocalBroadcastManager;
+
 import com.jasonette.seed.Core.JasonViewActivity;
 import com.jasonette.seed.Helper.JasonHelper;
 import com.jasonette.seed.Helper.JasonImageHelper;
+import com.github.florent37.singledateandtimepicker.dialog.SingleDateAndTimePickerDialog;
+import com.google.android.material.snackbar.Snackbar;
 
 import org.json.JSONArray;
 import org.json.JSONObject;
 
 import java.util.ArrayList;
 import java.util.Calendar;
 import java.util.Date;
-import java.util.GregorianCalendar;
+
+//import android.support.design.widget.Snackbar;
```

Comment:
If we don't need these, should we remove them?

Reply:
I removed this... The changes are madein develop branch

Reply:
Yes. They can be removed. Had commented just in case I had to roll back suddenly.

Reply:
Removed